Xamarin.Forms.Core/Items/ScrollToRequestEventArgs.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace Xamarin.Forms
+{
+	public class ScrollToRequestEventArgs : EventArgs
+	{
+		public ScrollToMode Mode { get; }
+
+		public ScrollToPosition ScrollToPosition { get; }
+		public bool Animate { get; }
```

Comment:
Verbs are best for method names. I'd prefer `Animated` or `IsAnimated`.

Reply:
Agreed. Renaming to `IsAnimated`.